llvm/lib/IR/Verifier.cpp

```diff
@@ -2231,6 +2231,27 @@ void Verifier::verifyFunctionAttrs(FunctionType *FT, AttributeList Attrs,
   checkUnsignedBaseTenFuncAttr(Attrs, "patchable-function-prefix", V);
   checkUnsignedBaseTenFuncAttr(Attrs, "patchable-function-entry", V);
   checkUnsignedBaseTenFuncAttr(Attrs, "warn-stack-size", V);
+
+  if (Attrs.hasFnAttr("sign-return-adress")) {
+    StringRef S = Attrs.getFnAttr("sign-return-adress").getValueAsString();
```

Comment:
Can do getFnAttr and check if it's valid rather than looking it up twice

Reply:
Done